ui/components/onboarding/TeamImageGenerator.tsx

```diff
@@ -40,7 +40,7 @@ export function ImageGenerator({ currImage, setImage, nextStage, stage }: any) {
         <FileInput file={inputImage} setFile={setInputImage} />
         {currImage || inputImage ? (
           <StageButton className="" onClick={submitImage}>
-            Save Design
+            Next
```

Comment:
Maybe I missed it - I can't find it anywhere else in the PR, did you move this button to be under the generated image?